### ISSUE TITLE
Error in outputs when removing folders

### ISSUE THREAD

**Issue by morgante** [terraform-google-modules/terraform-google-folders#16]
If you remove a folder from the list, it seems to introduce an error. I suspect this is because the resource still exists but the output is being interpolated.

Sample code: https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/terraform/test-org/org/folders.tf#L17

Error:
```
Error: Error in function call

  on .terraform/modules/folders-root/terraform-google-modules-terraform-google-folders-0c75f1c/outputs.tf line 19, in output "names_and_display_names":
  19:   value       = zipmap(var.names, google_folder.folders.*.display_name)
    |----------------
    | google_folder.folders is tuple with 2 elements
    | var.names is list of string with 1 element

Call to function "zipmap" failed: number of keys (1) does not match number of
values (2).
```

Likely fix is simply to only grab the first n google_folder resources for the zipmap.

**Comment by ludoo** [comment]
Yup, seen it happen in other places too, my usual workaround is to comment out outputs until things settle (which admittedly is not always possible).

We could remove `zipmap`s here and in the other multiple-resource modules, that would move the issue downstream in user's code but fix the issue at module levels.

**Comment by ludoo** [comment]
@morgante shall we keepp this open? I don't see a good way of fixing this in a non-disruptive way.

**Comment by morgante** [comment]
I think I have a fix.

**Comment by morgante** [comment]
@ludoo Turns out it was a simple fix: #20

**Comment by ludoo** [comment]
Simple and elegant, but I think it gives incorrect outputs in some of the cases it aims at fixing. Read my comment on the PR.
